packages/elements/src/components/ui/collapsible/CollapsibleContent.tsx

```diff
@@ -58,6 +54,10 @@ export const CollapsibleClickableContent: React.FC<CollapsibleClickableContentPr
   );
 };
 
-export const CollapsibleGroupHeading: React.FC<CollapsibleSimpleContentProps> = props => (
-  <CollapsibleSimpleContent className={styles.groupHeading} {...props} />
+export const CollapsibleGroupHeading: React.FC<CollapsibleSimpleContentProps &
+  HTMLAttributes<HTMLDivElement>> = ({ className, ...props }) => (
+  <CollapsibleContent
+    className={`${styles.groupHeading} ${className ?? ""}`}
```

Comment:
cx?

Reply:
If we want that dependency. The logic is currently simple enough, making me feel that cx is too much.

Reply:
I already included cx in the elements package for the button